config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
+
+## Kernel-configuration
+
+Used by VM-based runtimes only.
+
+* **`path`** (string, required) specifies an absolute path to the kernel
+  used to boot the virtual machine.
+* **`parameters`** (string, optional) specifies a space-separated list of parameters to pass to the kernel.
+* **`initrd`** (string, optional) specifies an absolute path to an initial ramdisk to be used by the kernel.
```

Comment:
More absolute path stuff here (both `path` and `initrd`).  Why require absolute paths?